src/wallet/rpcwallet.cpp

```diff
@@ -49,6 +49,10 @@ extern UniValue TxJoinSplitToJSON(const CTransaction& tx);
 int64_t nWalletUnlockTime;
```

Comment:
The file contains a mix of tabs and spaces. Please take the time to format it to spaces

Reply:
ok